src/plugins/pendo/components/OptInModal/OptInModalBody.tsx

```diff
@@ -33,16 +35,21 @@ export const OptInModalBody: React.FC<OptInModalProps> = ({ onContinue }) => {
   const borderColor = useColorModeValue('gray.100', 'gray.750')
 
   const CONSENT_TAG = `pendo_${getConfig().REACT_APP_PENDO_CONSENT_VERSION}`
-  const consent = VisitorDataManager.checkConsent(CONSENT_TAG)
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const consent = useMemo(() => VisitorDataManager.checkConsent(CONSENT_TAG), [])
```

Comment:
This is a string, we're safu
```suggestion
  const consent = useMemo(() => VisitorDataManager.checkConsent(CONSENT_TAG), [CONSENT_TAG])
```